sos/report/plugins/kdump.py

```diff
@@ -124,4 +124,42 @@ def setup(self):
         if self.get_option("collect-kdumps"):
             self.add_copy_spec(["/var/kdump-*"])
 
+
+class AzureKDump(KDump, AzurePlugin):
+
+    files = ('/etc/kdump.conf',)
+    packages = ('kexec-tools',)
+
+    def read_kdump_conffile(self):
+        """ Parse /etc/kdump file """
+        path = "/var/crash"
+
+        kdump = '/etc/kdump.conf'
+        with open(kdump, 'r', encoding='UTF-8') as file:
+            for line in file:
+                if line.startswith("path"):
```

Comment:
Does this mean that azure kexec/kdump can only dump the vmcores into already mounted filesystems?

Reply:
It dumps the vmcores into the rootfs by default; we wanted to start with this approach for now (by only reading the path variable in the /etc/kdump.conf file)